eth-bytecode-db/eth-bytecode-db/src/search/mod.rs

```diff
@@ -0,0 +1,5 @@
+mod bytecodes_comparison;
+mod candidates;
+mod partial_match;
+
+pub use partial_match::{find_partial_match_contract, BytecodeRemote, PartialMatchContract};
```

Comment:
Would the full match search be added further? I thought it should be a part of the same `search` function to look for the full match first, and then for the partial match if the previous was not successful

Reply:
I didn't add logic of full match, so I name this function "partial match". 
To be honest, I thought we would only have a partial match search, in next PR I can add `full_match.rs` search and combine it with `partial_match.rs` in some `any_search.rs`